front-end/src/views/KeyMeasuresPage/KeyMeasuresHome.vue

```diff
@@ -60,6 +84,45 @@
         if(measure.status && measure.status !== 'done' && deadline) {
           return new Date(deadline) < new Date();
         }
+      },
+      async submit() {
+        let measuresHtml = '';
+        this.keyMeasures.forEach(measure => {
+          measuresHtml += `<p><b>${measure.shortTitle} :</b></p>`;
+          measure.subMeasures.forEach(subMeasure => {
+            measuresHtml += `<p>${subMeasure.shortTitle} : ${subMeasure.status || ''}</p>`
+          });
+        });
+
+        const requestOptions = {
+          method: "POST",
+          headers: { "Content-Type": "application/json", "api-key": process.env.VUE_APP_SENDINBLUE_API_KEY },
+          body: JSON.stringify({
+            sender: { email: process.env.VUE_APP_SENDER_EMAIL, name: "site web ma cantine" },
+            to: [{ email: process.env.VUE_APP_CONTACT_EMAIL }],
+            replyTo: { email: process.env.VUE_APP_CONTACT_EMAIL },
+            subject: "Nouveau Béta-testeur ma cantine",
+            htmlContent: `<!DOCTYPE html> <html> <body>` +
+                         `<p><b>Cantine:</b> ${this.form.school}</p>` +
```

Comment:
Est-ce qu'il y a un autre façon de générer du HTML ? Ma concerne principale c'est d'ajouter n'importe quoi que l'utilisateur a saisi comme HTML

Reply:
J'ai suivi l'exemple de la doc : https://developers.sendinblue.com/reference#sendtransacemail
Quel risque tu vois ?

Reply:
On peut, par exemple, ajouter un lien (e.g. dans un input `<a href="https://dangerous-link.com>Click me!</a>`) qui quelqu'un sur notre équipe peut cliquer. On peut installer [cet library](http://underscorejs.org/#escape), qui fourni plein des methods utiles, ou simplement remplacer les mêmes characters qu'il remplace (`&, <, >, ", `, and '`)